Gemfile

```diff
@@ -3,6 +3,8 @@
 source "https://rubygems.org"
 
 gem "json"
+# For JSON Schema validation
+gem "json_schemer", "~> 2.4"
```

Comment:
please add to gemspec

Reply:
Ah right, I'm doing almost no gem development so totally forgot about that 
🫡